pipelined/bevy_pbr2/src/light.rs

```diff
@@ -48,6 +49,7 @@ impl Default for PointLight {
             intensity: 800.0, // Roughly a 60W non-halogen incandescent bulb
             range: 20.0,
             radius: 0.0,
+            shadows_enabled: true,
```

Comment:
I think we might want lights to have shadow casting disabled by default, given how expensive it is and how frequently people will reach for point lights (vs how frequently they would want shadows enabled, knowing the costs). Enabling shadows should be a conscious decision imo. And theres precedent for this, as both Unity and Godot default to shadows being off (for both light types). But I think its especially important to start disabled for PointLights.